react-apps/self-care-portal-frontend/toolkit/src/services/cdsUtils.js

```diff
@@ -40,9 +41,82 @@ export const convertSecondsToDaysHoursMinutes = (seconds) => {
 };
 
 export const getUserId = (user) => {
-  return user.email.endsWith('@carbon.super') ? user.email : user.email + '@carbon.super';  
-}; 
+  return user.email.endsWith('@carbon.super') ? user.email : user.email + '@carbon.super';
+};
 
 export const getTimeStamp = (timestamp) => {
   return moment(timestamp * 1000).format(dataTypes.daysHours);
 };
+
+export function generatePDF(consent, applicationName, consentStatus) {
+
+  const pdf = new jsPDF("p", "mm", "a4");
+  pdf.setFontSize(11);
+  pdf.rect(10, 10, 190, 275);
+
+  const permissionCategories = document.getElementsByClassName('clusterLabelText');
+  const permissions = document.getElementsByClassName('permissionsUL');
+  const accountIDs = document.getElementsByClassName('permittedAccount');
+
+  let contents = [];
+  let accounts = [];
+
+  try {
+
+    for (let i = 0; i < permissions.length; i++) {
+      let permissionTexts = permissions[i].getElementsByClassName('permissionText');
+      let permissionTextsJoined = "";
+      for (let j = 0; j < permissionTexts.length; j++) {
+        permissionTextsJoined += permissionTexts[j].innerHTML;
+        if (j < permissionTexts.length - 1) {
+          permissionTextsJoined += ", ";
+        }
+      }
+      contents.push(permissionTextsJoined);
+    }
+
+    for (let i = 0; i < accountIDs.length; i++) {
+      accounts.push(accountIDs[i].innerHTML);
+    }
+
+  } catch (e) {
+  }
+
+  pdf.text(20, 20, 'Consent ID : ' + consent.consentId);
+  pdf.text(20, 30, "Status : " + consentStatus);
```

Comment:
Why this use `"` while other lines use `'` ?

Reply:
Fixed with [32c32e1](https://github.com/wso2/reference-implementation-consumerdatastandards-au/pull/27/commits/32c32e1739493ddfd87cc473b436339cbaa48ba3)